index.html

```diff
@@ -0,0 +1,187 @@
+        <button
+          class="seeButton"
+          type="button"
+          onclick="location.href='/items'"
+        >
+          구경하러 가기
+        </button>
+      </div>
+      <img
+        src="src/shared/img/banner/banner01.svg"
+        class="bannerImg"
+        alt="bannerImg01"
+      />
+    </section>
+
+    <!-- 배너 02 -->
+    <section class="bannerLeft">
+      <div class="bannerContent">
+        <img
+          class="bannerImg02"
+          src="src/shared/img/banner/banner02.svg"
+          alt="bannerImg"
+        />
+        <div>
+          <p class="sectionLabel">Hot Item</p>
+          <h2 class="bannerTitle">
+            인기 상품을<br />
+            확인해 보세요
+          </h2>
+          <p class="bannerSubTitle">
+            가장 HOT한 중고거래 물품을<br />
+            판다마켓에서 확인해 보세요
+          </p>
+        </div>
+      </div>
+    </section>
+
+    <!-- 배너 03 -->
+    <section class="bannerRight">
+      <div class="bannerContent">
+        <div>
+          <p class="sectionLabel">Search</p>
+          <h2 class="bannerTitle">구매를 원하는 <br />상품을 검색하세요</h2>
+          <p class="bannerSubTitle">
+            구매하고 싶은 물품은 검색해서<br />
+            쉽게 찾아보세요
+          </p>
+        </div>
+        <img
+          class="bannerImg02"
+          src="src/shared/img/banner/banner03.svg"
+          alt="bannerImg"
+        />
+      </div>
+    </section>
+
+    <!-- 배너 04 -->
+    <section class="bannerLeft">
+      <div class="bannerContent">
+        <img
+          class="bannerImg02"
+          src="src/shared/img/banner/banner04.svg"
+          alt="bannerImg"
+        />
+        <div>
+          <p class="sectionLabel">Register</p>
+          <h2 class="bannerTitle">
+            판매를 원하는<br />
+            상품을 등록하세요
+          </h2>
+          <p class="bannerSubTitle">
+            어떤 물건이든 판매하고 싶은 상품을<br />
+            쉽게 등록하세요
+          </p>
+        </div>
+      </div>
+    </section>
+
+    <!-- 배너 05 -->
+    <section class="banner">
+      <div class="bannerContainer">
+        <h1 class="bannerTitle">
+          믿을 수 있는<br />
+          판다마켓 중고 거래
+        </h1>
+      </div>
+      <img
+        src="src/shared/img/banner/banner05.svg"
+        class="bannerImg"
+        alt="bannerImg05"
+      />
+    </section>
+
+    <footer class="footer">
+      <div class="footerContainer">
+        <p>©codeit - 2024</p>
+        <div class="buttonContainer">
+          <button type="button" onclick="location.href='/privacy'">
+            Privacy Policy
+          </button>
+          <button type="button" onclick="location.href='/faq'">FAQ</button>
+        </div>
+        <div class="snsButtonContainer">
+          <a
+            href="https://www.facebook.com/"
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            <img
+              src="src/shared/img/logo/sns/ic_facebook.svg"
+              alt="facebookButton"
+              class="snsLogo"
+            />
+          </a>
```

Comment:
💊 제안  
이 영역은 로고들의 모음이라고 할 수 있습니다. 관련된 요소들끼리 묶인 목록이니 의미록적인 마크업을 작성한다면 list 관련 태그로 작성하시는 것을 추천드립니다.

```suggestion
        <ul class="snsButtonContainer">
          <li>
            <a
              href="https://www.facebook.com/"
              target="_blank"
              rel="noopener noreferrer"
            >
              <img
                src="src/shared/img/logo/sns/ic_facebook.svg"
                alt="facebookButton"
                class="facebook"
              />
            </a>
          </li>
```